packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -20,6 +24,12 @@ export interface IScrollablePaneState {
 const getClassNames = classNamesFunction<IScrollablePaneStyleProps, IScrollablePaneStyles>();
 
 export class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, IScrollablePaneState> implements IScrollablePane {
+  private static readonly _scrollbarHeightKey = 'ScrollablePaneBase_ScrollbarHeight';
+  private _scrollLeft: number;
+  private _scrollTop: number;
+  private _isMounted: boolean;
+  private _listeningToEvents: boolean;
+  private _userInteractionStarted: boolean;
```

Comment:
Is this boolean necessary? Can't this be inferred by looking at scroll position or similar? All these booleans increase code path complexity.

Reply:
1. _scrollTop stores the [container.scrollTop](https://github.com/OfficeDev/office-ui-fabric-react/pull/9525/files?file-filters%5B%5D=.snap&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#diff-5a14aa84e3c90584e270c5e8307aa9abR558). Calculating [conatiner.scrollTop](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx#L273) may cause reflow.
2. _isMounted and _listeningToEvents are used to make sure there is no event listener or mutation observer if there is no Sticky component being used inside ScrollablePane. 
3. _userInteractionStarted is needed to know if user has started scrolling. This helps to skip calculations before PLT. _scrollTop wouldn't server the purpose as we won't be able to distinguish if user has scrolled back to top or user has not started scrolling. I don't want to keep _scrollTop === undefined check for the same.